src/Validation/Validation.php

```diff
@@ -1021,6 +1023,74 @@ public static function uploadedFile($file, array $options = [])
     }
 
     /**
+     * Validates a geographic coordinate.
+     *
+     * Supported formats are right now:
+     *
+     * - <latitude>, <longitude> Example: -25.274398, 133.775136
+     *
+     * ### Options
+     *
+     * - `type` - A string of the coordinate format, right now only `latLong`.
```

Comment:
Feel free to change it I've added you as contributer and @markstory as well to my fork.

I'm busy today going to leave work a work related trip and will be back the 17th.